src/components/editor/Editor.jsx

```diff
@@ -61,7 +61,9 @@ class Editor extends Component {
   }
 
   // The ld4p group is only for templates
-  groupsToSaveInto = () => Config.groupsInSinopia.filter(group => group[0] !== 'ld4p')
+  groupsToSaveInto = () => Object.entries(Config.groupsInSinopia)
```

Comment:
i ripped off the `Object.entries` approach from the sinopia_acl codebase, since that's where i got the new config structure.